### PR TITLE
mes 6216 add scanrepo and gitsecrets

### PR DESCRIPTION
**Pull request by piercemorris**
## Description

Installed husky to enable git hooks in package.json with both pre-commit & pre-push hooks implemented. Instructions added for installing both git secrets & scan repo to local machines.

## Checklist

- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Code has been tested manually
- [x] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -1 +1,4 @@
-node_modules
\ No newline at end of file
+node_modules
+mock
```
Comment (ammarhaiderbjss) [inline .eslintignore:2]: I think we might want to check in the mock files as they're used in order to serve a local journal

**Comment by ammarhaiderbjss** [comment]
Looks good, assuming we've run the hooks to test they work?

File: .eslintignore (hunk 1 above)
Comment (piercemorris) [reply .eslintignore:2]: This is just ignoring them in the linting, only the files in the src/ folder will be linted

**Comment by piercemorris** [comment]
> Looks good, assuming we've run the hooks to test they work?

Yes they work, however I'm trying to find a case where they fail either git secrets or scan repo

File: .eslintignore (hunk 1 above)
Comment (ammarhaiderbjss) [reply .eslintignore:2]: ah of course 👍

**Review by ammarhaiderbjss — APPROVED** [review]

**Review by szabikr — APPROVED** [review]

**Pull request merged**